kfp/README.md

```diff
@@ -4,20 +4,23 @@
 
 | Transform                           |                                    KFP pipeline                                    |          
 |-------------------------------------|:----------------------------------------------------------------------------------:|
-| code/malware                        |                  [malware_wf.py](../transforms/code/malware/kfp_ray/v1/malware_wf.py)                   |
-| code/code_quality                   |            [code_quality_wf.py](../transforms/code/code_quality/kfp_ray/v1/code_quality_wf.py)            |
-| code/programming language_annotator | [proglang_select_wf.py](../transforms/code/proglang_select/kfp_ray/v1/proglang_select_wf.py) |
-| universal/doc_id                    |                  [doc_id_wf.py](../transforms/universal/doc_id/kfp_ray/v1/doc_id_wf.py)                   |
-| universal/ededup                    |                  [ededup_wf.py](../transforms/universal/ededup/kfp_ray/v1/ededup_wf.py)                   |
-| universal/fdedup                    |                  [fdedup_wf.py](../transforms/universal/fdedup/kfp_ray/v1/fdedup_wf.py)                   |
-| universal/filtering                 |              [filter_wf.py](../transforms/universal/filter/kfp_ray/v1/filter_wf.py)              |
-| universal/noop                      |                     [noop_wf.py](../transforms/universal/noop/kfp_ray/v1/noop_wf.py)                      |
-| universal/tokenization              |         [tokenization_wf.py](../transforms/universal/tokenization/kfp_ray/v1/tokenization_wf.py)          |
+| language/lang_id                  |                  [lang_id_wf.py](../transforms/language/lang_id/kfp_ray/lang_id_wf.py)                   |
```

Comment:
This large list will be hard to keep in sync. Do we really want it here or can we link to the transforms README maybe?